charts/forms-flow-web/templates/configmap.yaml

```diff
@@ -59,5 +59,7 @@ data:
         "ENABLE_TASKS_MODULE": "${ENABLE_TASKS_MODULE}",
         "ENABLE_DASHBOARDS_MODULE": "${ENABLE_DASHBOARDS_MODULE}",
         "ENABLE_PROCESSES_MODULE": "${ENABLE_PROCESSES_MODULE}",
-        "ENABLE_APPLICATIONS_MODULE": "${ENABLE_APPLICATIONS_MODULE}"
+        "ENABLE_APPLICATIONS_MODULE": "${ENABLE_APPLICATIONS_MODULE}",
+        "REACT_APP_ENABLE_CHATBOT": "${REACT_APP_ENABLE_CHATBOT}",
+        "REACT_APP_CHATBOT_URL": "${REACT_APP_CHATBOT_URL}"
```

Comment:
Do we need the chatbot env here ? Does this has ee image

Reply:
Its Just env that we have provided here. If we are using EE it will be useful